### PR TITLE
Manual backport of #11145 (Use proper precision when format not specified)

### PR DESCRIPTION
**Pull request by pllim**
Use proper precision when format not specified

### Description

This pull request is to manually backport #11145 for 4.2.1.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1152,6 +1152,9 @@ astropy.units
 - Ensure ``keepdims`` works for taking ``mean``, ``std``, and ``var`` of
   ``Quantity``. [#11198]
 
+- For ``Quantity.to_string()``, ensure that the precision argument is also
```
Comment (pllim) [inline CHANGES.rst:1155]: Do I have to move the change log as part of the manual backport?

Comment (mhvk) [reply CHANGES.rst:1155]: I would think merging is at the time of the release, but am not sure...

**Review by mhvk — APPROVED** [review]
Looks good except I don't know the answer to the question about the changelog. I *think*, though, it should just be backported as is.

**Comment by eteq** [comment]
I think it's right to leave the changelog as it is here. I'm not going to worry too much about it since towncrier will make this all mott anyway in future versions!

**Pull request merged**